.github/workflows/test-cluster-set-up.yml

```diff
@@ -0,0 +1,70 @@
+name: Create testing cluster
+
+# Make sure to pass the distribution type and security feature as parameters
+# { 
+#  "distribution": "RPM/DEB/TAR",
+#  "security": "enable/disable"
+# }
```

Comment:
Update the comments and mention specific this is for payload passing, maybe giving an example or some sort.

Reply:
Done!

Reply:
Resolved.